default.yml

```diff
@@ -46,17 +46,15 @@ Layout/MultilineOperationIndentation:
 Lint/SuppressedException:
   Enabled: false
 
-Layout/SpaceInsideBlockBraces:
-  # The space here provides no real gain in readability while consuming
-  # horizontal space that could be used for a better parameter name.
-  # Also {| differentiates better from a hash than { | does.
-  SpaceBeforeBlockParameters: false
-
```

Comment:
Please skip this one but add the `Style/HashEachMethods` change

Reply:
Done!